locations.md

```diff
@@ -6,13 +6,19 @@
 
 | Field           | Type     | Description                                     |
 | --------------- | -------- | ----------------------------------------------- |
-| `id`|bigint|the unique identifier|
-| `name`|string|the name of the location|
-| `country`|string|the country of the location|
+| `id`|int|the unique identifier|
 | `type`|string|the type of location (`market`, `port`, or `zip3`)|
+| `geographyType`|string|the type of geography (`circle` or `none`)|
+| `date`|timestamp|time and date of location creation|
+| `name`|string|the name of the location|
+| `state`|string|the state or province of the location, if applicable|
+| `countryId`|int|the unique identifier of the country
```

Comment:
we should omit countyId we don't want people relying on it and in the docs we state to to rely on undocumented api fields